qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -43,6 +43,7 @@
     variants:
         - gagent_install:
             gagent_check_type = install
+            locate_qga_rpm_cmd = "ls /qemu-guest-agent*"
```

Comment:
This is not fully automated, right? You manually put the rpm package here before the test start?

Reply:
no, the qga rpm packages were installed and downloaded locally,and these operations is executed by internal .
It's my fault , I should @you in gitlab for another part of this patch.

Reply:
@6-dehan If people has no your internal tool, how should they test with this script? Better give a instruction with comment and also I recommend you have the rpm location instead of the command as the variable.

Reply:
Recommend:
1. Name it `qga_rpm_path`.
2. Allow 2 ways to define it: a. Provide path of the  directly; b. A shell command to get it like here you have.
3. Allow it to be NULL, in this case you do nothing for the installation.

To implement 2,
`qga_rpm_path = cmd:ls /qemu-guest-agent*`
```
qga_rpm_path = params.get(qga_rpm_path')
if qga_rpm_path and qga_rpm_path.startswith('cmd:'):
qga_rpm_path = session.cmd_output_safe(qga_rpm_path.split(':')[1])
```